internal/plugin/adapter/resource.go

```diff
@@ -39,13 +39,13 @@ func NewResource[T any](
 
 type resourceAdapter[T any] struct {
 	name         string
-	view         View[T]
+	view         ResView[T]
```

Comment:
I guess we need to implement `ConfigValidators` as well as in datasource

Reply:
That's correct. I'll fix this with the next PR.